caddyhttp/compress/zstd.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015 Light Code Labs, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compress
+
+import (
+	"io/ioutil"
+	"io"
+	"sync"
+
+	"github.com/DataDog/zstd"
```

Comment:
This library is using `cgo`, I don't like it...

99,8 % of the code is C!!!!!!

Reply:
There will be cross compile issues!!!!